libr/core/graph.c

```diff
@@ -2036,6 +2036,7 @@ R_API int r_core_visual_graph(RCore *core, RAnalFunction *_fcn, int is_interacti
 		goto err_graph_new;
 	}
 	g->movspeed = r_config_get_i (core->config, "graph.scroll");
+	g->invscroll = r_config_get_i (core->config, "graph.invscroll");
```

Comment:
Also, if you move the r_config_get_i inside the while loop you can get the change in the scroll direction without having to exit the ascii-art graph mode.